edx_when/__init__.py

```diff
@@ -2,4 +2,4 @@
 Central source of course block dates for the LMS.
 """
 
-__version__ = '2.5.2'
+__version__ = '2.5.3'
```

Comment:
This isn't a bugfix, it should be a minor version update.

Reply:
it' already minor version update, do you want to say subversion?

Reply:
@feanil ⬆️

Reply:
Version numbers are MAJOR.MINOR.PATCH according to [semver](https://semver.org/) so given that this is not a bug fix. It should be the middle version number that gets bumped and the last version number should be zeroed. So `2.6.0` in this case.